src/lib.js

```diff
@@ -677,6 +676,36 @@ var program,
       }
     },
 
+    /**
+     *
+     * @param {string} dataString - Input data string to check if it is a graphql query
+     * @returns {Object} - { result: true, graphql: {Object} } if dataString is a graphql query else { result: false }
+    */
+    checkIfGraphql: function (dataString) {
```

Comment:
Function naming suggests value to be of type bolean. i.e. Answer to question like `checkIfGraphql` can be true or false only.

Can we rename this better to suggest what it actually does?

Reply:
Changed `checkIfGraphql` to `identifyGraphqlRequest`.